rollup/pipeline/pipeline.go

```diff
@@ -130,7 +130,10 @@ func (p *Pipeline) Start(deadline time.Time) error {
 
 // Stop forces pipeline to stop its operation and return whatever progress it has so far
 func (p *Pipeline) Stop() {
-	close(p.txnQueue)
+	if p.txnQueue != nil {
+		close(p.txnQueue)
+		p.txnQueue = nil
```

Comment:
we still need to nullify txnQueue because ScrollWorker might try to push txns [here](https://github.com/scroll-tech/go-ethereum/blob/69aae68b2f49dab456e7f116858befb48c43f97e/miner/scroll_worker.go#L359-L361) regardless of pipeline being stopped or not.